core/dbt/materializations/incremental/microbatch.py

```diff
@@ -35,10 +36,13 @@ def __init__(
             event_time_start.replace(tzinfo=pytz.UTC) if event_time_start else None
         )
         self.event_time_end = event_time_end.replace(tzinfo=pytz.UTC) if event_time_end else None
+        self.batch_current_time = (
+            batch_current_time if batch_current_time is not None else datetime.now(pytz.UTC)
+        )
```

Comment:
Nit: can be simplified to 

```suggestion
        self.batch_current_time = (
            batch_current_time or datetime.now(pytz.UTC)
        )
```